aip/general/0193.md

```diff
@@ -69,13 +69,27 @@ and the method **should** put partial failure information in the metadata
 message. The errors themselves **must** still be represented with a
 [`google.rpc.Status`][Status] object.
 
+### Permission Denied
+
+If the API requires authentication and authorization, the API **must** first
+check whether the user has the permission to perform the operation, and **must**
+error with `PERMISSION_DENIED` (HTTP 403) if so.
```

Comment:
I think this could be split into two sentences.

Also phrasing "if so" at the end of the sentence would imply the user has the correct permission in which case the error should not be returned.

Reply:
@GregFurmanek IIUC, this verbiage was brought over directly from each of the other AIPs. If we want to change it, I would prefer to do that in separate PR.

Reply:
In that case let's use as is and we will update it separately.

Reply:
Oh wait, scratch that. It was changed.

@toumorokoshi sometimes less is more--leaving some interpretation allowed--is desirable in AIPs. I think this is one of those cases. Can we simply move the repeated copy as is for now? Is there anything specifically required for Uniform Resources that necessitates a change to the existing copy?

Reply:
no, I just like adding context.

I'll revert to the original wording.